climpyrical/cmd/preprocess_model.py

```diff
@@ -0,0 +1,169 @@
+"""
+quick usage of climpyrical preprocess_mode.py
+usage:
+python preprocess_model.py -i input.nc -o output.nc -m True
+
+This script takes a CanRCM4 model at the native resolution and
+downscales from 50 km to  5 km and fills in missing land values
+using external masks.
+"""
+
+from climpyrical.data import read_data, gen_dataset, interpolate_dataset
+from climpyrical.gridding import regrid_ensemble, extend_north
+
+import click
+from pkg_resources import resource_filename
+import logging
+
+import warnings
+
+import numpy as np
+
+warnings.filterwarnings("ignore")
+
+
+@click.command()
+@click.option("-i", "--in-path", help="Input CanRCM4 file", required=True)
+@click.option("-o", "--out-path", help="Output file", required=True)
+@click.option(
+    "-m", "--fill-glaciers", help="Refill glacier points", default=True
+)
+@click.option(
+    "-l",
+    "--log-level",
+    help="Logging level",
+    type=click.Choice(["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]),
+    default="INFO",
+)
+def run_processing(in_path, out_path, fill_glaciers, log_level):
```

Comment:
This name (and the subsequent description) is very generic and nearly meaningless. It's basically "this does stuff that needs to be done". Which begs the question... what exactly does it do? Can you try to be more specific about what constitutes "processing"? What are the specific actions that it takes?

Reply:
Sure, some more specificity can be found at the [top of the file](https://github.com/pacificclimate/climpyrical/blob/0de7ca1455071346a0bd645cbcd2625bb3328a94/climpyrical/cmd/preprocess_model.py#L6). This should go in `run_processing()` because that's clearly the first place you (and probably anyone else) will look.